target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -613,6 +613,16 @@ define Device/tplink_tl-wr810n-v2
 endef
 TARGET_DEVICES += tplink_tl-wr810n-v2
 
+define Device/tplink_tl-wr841hp-v3
+  $(Device/tplink-8mlzma)
+  SOC := qca9533
+  DEVICE_MODEL := TL-WR841HP
+  DEVICE_VARIANT := v3
+  TPLINK_HWID := 0x08411003
+  SUPPORTED_DEVICES += tl-wr841hp-v3
```

Comment:
This line can be dropped.

Reply:
It will look like this ?
```
define Device/tplink_tl-wr841hp-v3
  $(Device/tplink-8mlzma)
  SOC := qca9533
endef
TARGET_DEVICES += tplink_tl-wr841hp-v3

```

Reply:
Just drop the SUPPORTED_DEVICES _line_.